reana_client/cli/files.py

```diff
@@ -140,8 +141,19 @@ def get_files(ctx, workflow, _filter,
     help='Path to the directory where files will be downloaded.')
 @add_access_token_options
 @click.pass_context
-def download_files(ctx, workflow, filenames, output_directory, access_token):
-    """Download workflow workspace file(s)."""
+def download_files(ctx, workflow, filenames,
+                   output_directory, access_token):  # noqa: D301
+    """Download workspace files.
+
+    This command permits to download workspace files. By default, the files
```

Comment:
Please standardise everywhere the first sentence to:

    The `download` command allows to...
    The `ls` commands lists...   
    The `start` command allows to...
    The `status` command retrieves...
    ...

Reply:
Also there is `workflow.N` still in many places instead of myanalysis.42:

```console
$ git grep 'workflow\.[0-9]'
docs/gettingstarted.rst:   workflow.1
docs/gettingstarted.rst:   $ reana-client status -w workflow.1
docs/gettingstarted.rst:   $ export REANA_WORKON=workflow.1
docs/gettingstarted.rst:   workflow.1 has been started.
docs/userguide.rst:   workflow.1 has been started.
docs/userguide.rst:    $ reana-client stop --force --workflow=otherworkflow.1
reana_client/cli/files.py:    \t $ reana-client ls --workflow workflow.3
reana_client/cli/files.py:    \t $ reana-client upload -w workflow.2 code/mycode.py
reana_client/cli/files.py:    \t $ reana-client rm -w workflow.4 data/mydata.csv \n
reana_client/cli/workflow.py:    \t $ reana-client start -w workflow.6 -p sleeptime=10 \n
reana_client/cli/workflow.py:    \t $ reana-client start -w workflow.6 -p myparam1=myvalue1
reana_client/cli/workflow.py:    \t $ reana-client stop -w workflow.1 --force
reana_client/cli/workflow.py:    \t $ reana-client delete -w myworkflow.42 \n
reana_client/cli/workflow.py:    \t $ reana-client delete -w myworkflow.42 --include-records
reana_client/cli/workflow.py:    \t $ reana-client diff myworkflow.42 myotherworkflow.21 \n
reana_client/cli/workflow.py:    \t $ reana-client diff myworkflow.42 myotherworkflow.21 --brief
reana_client/utils.py:        Name might be in format 'reana.workflow.123' with arbitrary
tests/test_cli_workflows.py:            'name': 'workflow.1',
tests/test_cli_workflows.py:        'name': 'workflow.5',
tests/test_cli_workflows.py:                'name': 'workflow.19',
tests/test_cli_workflows.py:                ['start', '-t', reana_token, '-w workflow.19',
tests/test_cli_workflows.py:    workflow = "workflow.1"
```